src/platform/i386/chal_cpu.h

```diff
@@ -91,10 +92,11 @@ static void
 chal_cpu_init(void)
 {
 	u32_t cr4 = chal_cpu_cr4_get();
+	int cpu_id = get_cpuid();
```

Comment:
Use of a `cpuid_t` (not present yet) would be great imo!

Reply:
indeed